src/engine/engine.rs

```diff
@@ -26,8 +27,26 @@ pub trait Filesystem: HasName + HasUuid {
 }
 
 pub trait BlockDev: HasUuid {
-    /// path of the device node
+    /// Get the path of the device node for this device.
```

Comment:
Reiterating concern about ```user_id``` name. It does not seem to be an id, properly speaking.

Reply:
ok. let's try `user_info` and `hardware_info`.